website/pages/docs/platform/k8s/helm/configuration.mdx

```diff
@@ -121,13 +122,28 @@ and consider if they're appropriate for your deployment.
 
   - `priorityClassName` (`string: ""`) - Priority class for injector pods
 
+  - `annotations` (`dictionary: {}`) - This value defines additional annotations for injector pods. This can either be YAML or a YAML-formatted multi-line templated string.
+
+    ```yaml
+    annotations:
+      "sample/annotation1": "foo"
+      "sample/annotation2": "bar"
+    # or
+    annotations: |
+      "sample/annotation1": "foo"
+      "sample/annotation2": "bar"
+    ```
+
+  - `failurePolicy` (`string : ""`) - When set to `Fail`, an error calling the webhook causes the admission to fail and the API request to be rejected.
```

Comment:
What do you think about including a blurb here saying what the allowed values are? Something like 
```
Allowed values: `Fail`, `Ignore`
```

Reply:
Done!